products/internetexplorer.md

```diff
@@ -0,0 +1,50 @@
+---
+permalink: /internetexplorer
+title: Internet Explorer
+releasePolicyLink: https://docs.microsoft.com/en-us/lifecycle/faq/internet-explorer-microsoft-edge#what-is-the-lifecycle-policy-for-internet-explorer-
+releaseDateColumn: true
+releaseColumn: false
+iconSlug: internetexplorer
+activeSupportColumn: false
+eolColumn: Security and technical support
+sortReleasesBy: releaseDate
+releases:
+
+-   releaseCycle: "7" 
+    eol: 2023-10-10
+    releaseDate: 2006-10-18
+
+-   releaseCycle: "8"
+    eol: 2016-01-12
+    releaseDate: 2009-06-17
+
+    
+-   releaseCycle: "9"
+    eol: 2016-01-12
+    releaseDate: 2011-03-15
+
+-   releaseCycle: "10"
+    eol: 2020-01-31
+    releaseDate: 2012-10-30
+
+-   releaseCycle: "11 (Semi-Annual Channel)" # This applies to 99% of users
```

Comment:
[MS also calls it the desktop app](https://docs.microsoft.com/en-us/lifecycle/products/internet-explorer-11), which is much more understandable. 

Also remove the comment, it's not useful as a hidden comment.  
```suggestion
-   releaseCycle: "11 (desktop app)"
```

Reply:
The issue I see with this is that technically what is on the LTSC is also the desktop app.

Reply:
What's LTSC?

Reply:
> What's LTSC?

Long term support for Windows. This also applies for Windows Server desktop.